src/components/reader-funded/readerFundedSubscribeCard.tsx

```diff
@@ -76,21 +76,11 @@ const ReaderFundedSubscribeCard = (props: ReaderFundedSubscribeCardProps) => {
     ${until.tablet} {
       display: none;
     }
-
-    ${from.tablet} {
-      width: 100%;
-      margin: 0 auto;
-      padding-top: 31%;
-      background-size: contain;
-      background-repeat: no-repeat;
-      background-position: right top;
-      background-image: url(${props.imagePath});
-    }
   `;
 
   return (
     <div css={container}>
-      <div css={image} />
+      <img src={props.imagePath} css={image} />
```

Comment:
We should have an `alt` attribute on an `<img>` element. A case could be made that these images are just decorative, in which case we could just add an empty alt attribute `alt=""`, or otherwise add an altText prop to the component to be passed in along with the image source?

Reply:
Ah yes, I'd probably agree that these images are decorative and any alt e.g. "newspaper" would be redundant so will update this with an empty alt attribute